pkg/operator/info.go

```diff
@@ -15,7 +15,7 @@ import (
 func (o *Operator) GetInfo(args []string) error {
 	if len(args) != 1 {
 		color.Set(color.FgRed)
-		return errors.New("Invalid number of arguments. Usage: tasty info OPERATOR_NAME")
```

Comment:
Why not start a sentence with a capital letter?

Reply:
https://github.com/golang/go/wiki/CodeReviewComments#error-strings